cEP-0010.md

```diff
@@ -122,155 +120,161 @@ in snake_case).
+  taste represents the minimun number of token that have to be equal to be
+  considered as a clone.
+- `Root.Redundancy.UnusedImport`: This taste describes useless import/include
+  that is not needed.
+- `Root.Redundancy.UnreachableCode`: This taste describes source code that can
+  never be executed during the program execution.
+- `Root.Redundancy.UnusedFunction`: This taste describes functions that are
+  never called during code execution
+- `Root.Redundancy.UnreachableStatement`: This taste describes statement that
+  are never executed during code execution.
+- `Root.Redundancy.UnusedParameter`: This taste describes functions' parameters
+  which are never used.
+- `Root.Redundancy.UnusedLocalVariable`: This taste describes variables which
+  are defined locally but never used.
+- `Root.Redundancy.UnusedGlobalVariable`: this taste describes variables which
+  have a global scope but are never used.
+- `Root.Spelling`: This aspect checks on the spelling of specific words.
+- `Root.Spelling.coalaCorrect`: This aspect checks on the spelling of `coala`.
+- `Root.Spelling.aspectsYEAH`: This aspect checks on the spelling of `aspect[s]`
+  and `aspectsYEAH`.
+- `Root.UndefinedElement`: This aspect checks on undefined element in your
+  codebase.
+- `Root.UndefinedElement.UndefinedVariable`: This aspect detects undefined
+  variables in your code.
+- `Root.UndefinedElement.UndefinedParameter`: This aspect detects undefined
+  parameters in functions' definitions.
+- `Root.UndefinedElement.UndefinedFunction`: This aspect detects undefined
+  functions in your codebase.
+- `Root.Metadata`: This describes any aspect that is related to metadata that is
+  not inside your source code.
+- `Root.Metadata.CommitMessage`.
+- `Root.Metadata.CommitMessage.Emptiness`: This aspect is to detect and prevent
+  from using empty commit message.
+- `Root.Metadata.CommitMessage.Shortlog`: This aspect checks on the first line
+  of your commit message.
+- `Root.Metadata.CommitMessage.Shortlog.ColonExistence`: This aspect checks your
+  commit shortlog to see whether or not there is a colon (given that the usage
+  of colon is compulsory in your project).
+- `Root.Metadata.CommitMessage.Shortlog.TrailingPeriod`: This aspect checks for
+  trailing period.
+- `Root.Metadata.CommitMessage.Shortlog.Tense`: This aspect checks on the tense
+  used in commit message shortlog.
+- `Root.Metadata.CommitMessage.Shortlog.Length`: This aspect checks on the
+  length of your commit message shortlog given the value of
+  `Root.Metadata.CommitMessage.Shortlog.Length.max_shortlog_length`.
+- `Root.Metadata.CommitMessage.Shortlog.FirstCharacter`: This aspect detects
+  inconsistent casing for the first character of your commit message
+  shortlog(and the the first character following the colon if there is one)
+  given the value of
+  <!-- Ignore MarkdownBear -->
+  `Root.Metadata.CommitMessage.Shortlog.FirstCharacter.shortlog_starts_upper_case`
```

Comment:
MarkdownBear wants to warp inline codeblock, this is not necessary with `remark-lint@6`, but MarkdownBear is using v5.